flow/hash.go

```diff
@@ -38,6 +38,29 @@ func (fl *ICMPLayer) Hash(hasher hash.Hash) {
 	hasher.Write(value32)
 }
 
+func (tl *TransportLayer) Hash(hasher hash.Hash) {
+	if tl == nil {
+		return
+	}
+
+	valueA := make([]byte, 8)
+	valueB := make([]byte, 8)
+	binary.BigEndian.PutUint64(valueA, uint64(tl.A))
+	binary.BigEndian.PutUint64(valueB, uint64(tl.B))
+
+	if tl.A > tl.B {
+		hasher.Write(valueA)
+		hasher.Write(valueB)
+	} else {
+		hasher.Write(valueB)
+		hasher.Write(valueA)
+	}
+
+	valueID := make([]byte, 8)
+	binary.BigEndian.PutUint64(valueID, uint64(tl.ID))
```

Comment:
As A and B are 16bits (port number), I think it would be better (perfomances) to do

```
value32 := make([]byte, 4)
if t1.A > t1.B {
   binary.BigEndian.PutUint32(value32, uint32(t1.A<<16|t1.B))
} else {
   binary.BigEndian.PutUint32(value32, uint32(t1.B<<16|t1.A))
}
hasher.Write(value32)

valueID := make([]byte, 8)
binary.BigEndian.PutUint64(valueID, uint64(tl.ID))
```